src/skibidi_orm/migration_engine/revisions/constants.py

```diff
@@ -0,0 +1,18 @@
+from dotenv import load_dotenv
+
+
+"""Constants regarding revision managing, such as operations used to
+create revision tables, etc."""
+
+DEFAULT_REVISION_TABLE_NAME = "__revisions"
+
+
+def get_revision_table_name() -> str:
+    """Get the name of the table used to store revision data.
+    If no name is set via an environment variable, the default name is used."""
+    load_dotenv()
+
+    try:
+        return __CHUMPY_REVISION_TABLE_NAME  # type: ignore
+    except NameError:
+        return DEFAULT_REVISION_TABLE_NAME
```

Comment:
Not a good way to check if an environmental variable was set.

Use either:
```python
if "FOO" in os.environ:
    pass
```
or
```python
if os.getenv("FOO") is not None:
    pass
```